src/main/java/io/github/lambdatest/gradle/AppUploader.java

```diff
@@ -13,6 +13,9 @@ public class AppUploader {
     private String appFilePath;
 
     public AppUploader(String username, String accessKey, String appFilePath) {
+        if (username == null || accessKey == null || appFilePath == null)
```

Comment:
HI @Taz03 ,

Thank you for the pr.
Having a generic message is good, but thinking if we can improve more.  below is my sample suggestion, let me know your thoughts!
```suggestion
 if (username == null) throw new IllegalArgumentException("Username cannot be null");
 if (accessKey == null) throw new IllegalArgumentException("Access Key cannot be null");
 if (appFilePath == null) throw new IllegalArgumentException("App File Path cannot be null");
```

Thanks,
Sri!

Reply:
I feel we should go for this as well. It is more accurate.